docs/userguide/design.md

```diff
@@ -126,25 +123,22 @@ affect the original source service, or vice-versa.
 
 Thus, the normal mode of operation is to:
 
-1.  Open a service with a given locale.
-
-2.  Use the service as long as needed. However, do not keep opening and closing
-    a service within a tight loop.
-
-3.  Clone a service if it needs to be used in parallel in another thread.
+1. Open a service with a given locale.
+1. Use the service as long as needed. However, do not keep opening and closing
+   a service within a tight loop.
+1. Clone a service if it needs to be used in parallel in another thread.
+1. Close any clones that you open as well as any instances of the services that
+   are owned.
 
-4.  Close any clones that you open as well as any instances of the services that
-    are owned.
-
-*These service instances may be closed in any sequence. The preceding steps are
-given as an example.*
+:point_right: These service instances may be closed in any sequence. The
```

Comment:
Please use the agreed "note" style from the instructions doc, for all notes.

`> :point_right: **Note**: blah etc`

Reply:
Same as above - updated all occurrences.